### ISSUE TITLE
make rel_inets error

### ISSUE THREAD

**Issue by joshzinny** [nitrogen/nitrogen#113]
==> rel (generate)
ERROR: generate failed while processing /Users/joshzinny/nitrogen/rel: {'EXIT',{{badmatch,{error,"Application dialyzer uses non existing application wx"}},
         [{rebar_reltool,generate,2,
                         [{file,"src/rebar_reltool.erl"},{line,53}]},
          {rebar_core,run_modules,4,[{file,"src/rebar_core.erl"},{line,493}]},
          {rebar_core,execute,6,[{file,"src/rebar_core.erl"},{line,418}]},
          {rebar_core,maybe_execute,8,
                      [{file,"src/rebar_core.erl"},{line,302}]},
          {rebar_core,process_dir1,7,[{file,"src/rebar_core.erl"},{line,261}]},
          {rebar_core,process_commands,2,
                      [{file,"src/rebar_core.erl"},{line,93}]},
          {rebar,main,1,[{file,"src/rebar.erl"},{line,58}]},
          {escript,run,2,[{file,"escript.erl"},{line,758}]}]}}
make[3]: *** [Makefile:303: generate] Error 1
make[3]: Leaving directory '/Users/joshzinny/nitrogen'
make[2]: *** [Makefile:323: rel_inner_full] Error 2
make[2]: Leaving directory '/Users/joshzinny/nitrogen'
make[1]: *** [Makefile:213: rel] Error 2
make[1]: Leaving directory '/Users/joshzinny/nitrogen'
make: *** [Makefile:103: rel_inets] Error 2
bash: cd: ../myapp.: No such file or directory

**Comment by choptastic** [comment]
Hi, this is yelling because the `wx` application is missing.

Can I ask how you installed Erlang? It looks like your installation is missing some applications.

**Comment by joshzinny** [comment]
I used brew I think, and I'm running 22.0

**Comment by choptastic** [comment]
Hmm.  i'm not super well versed in running Erlang with OSX, but if you can, give the Erlang-Solutions installer a try: https://www.erlang-solutions.com/resources/download.html

It should install Erlang with everything.

**Comment by joshzinny** [comment]
Yes, I just uninstalled the brew and the mac port one. I have installed the one from erlang-solution but it doesn't install erlang in my system for some reason.

**Comment by choptastic** [comment]
Interesting.  Hard to say what's happening, but the ErlangInstaller from Erlang Solutions is really just that - an installer.  So when you install the installer and run it, it gives a little "E" icon on the menu bar that you can use to install specific versions of Erlang, or directly run an Erlang Shell.  It did take me a while to figure that out when I first used it.

Does that help, or are you already there and something still isn't working?

**Comment by choptastic** [comment]
Hi, just following up here.  Any luck with my previous suggestion?
